navbarBuild.css

```diff
@@ -0,0 +1,41 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	font-family: arial, sans-serif;
+	
+}
+
+nav {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	overflow: auto;
+	
+}
+
+
+nav ul {
+	margin: 0;
+	width: 100%;
+	padding: 0;
+	text-align: center;
```

Comment:
add overflow auto here

Reply:
Thanks Josh. Could you review the change to font sizing thanks.